ui/jetbrains/src/main/java/com/neueda/jetbrains/plugin/graphdb/jetbrains/ui/console/log/LogPanel.java

```diff
@@ -107,8 +119,11 @@ public void metadataRefreshSucceed(DataSourceApi nodeDataSource, DataSourceMetad
 
             @Override
             public void metadataRefreshFailed(DataSourceApi nodeDataSource, Exception exception) {
-                error(String.format("DataSource[%s] - metadata refresh failed. Reason: ", nodeDataSource.getName()));
-                printException(exception);
+                Map<String, String> exceptions = new HashMap<>();
```

Comment:
Is this needed?

Reply:
Yes, to avoid duplicated messages in the console.